docs/dev/table/common.md

```diff
@@ -345,133 +359,139 @@ tableEnv.registerTable("projectedTable", projTable)
 table_env = ... # see "Create a TableEnvironment" section
 
 # table is the result of a simple projection query 
-proj_table = table_env.scan("X").select(...)
+proj_table = table_env.from_path("X").select(...)
 
 # register the Table projTable as table "projectedTable"
 table_env.register_table("projectedTable", proj_table)
 {% endhighlight %}
 </div>
 </div>
 
-**Note:** A registered `Table` is treated similarly to a `VIEW` as known from relational database systems, i.e., the query that defines the `Table` is not optimized but will be inlined when another query references the registered `Table`. If multiple queries reference the same registered `Table`, it will be inlined for each referencing query and executed multiple times, i.e., the result of the registered `Table` will *not* be shared.
+**Note:** A `Table` object similarly to a `VIEW` from relational database
+systems, i.e., the query that defines the `Table` is not optimized but will be inlined when another
+query references the registered `Table`. If multiple queries reference the same registered `Table`,
+it will be inlined for each referencing query and executed multiple times, i.e., the result of the
+registered `Table` will *not* be shared.
 
 {% top %}
 
-### Register a TableSource
+#### Connector tables
 
-A `TableSource` provides access to external data which is stored in a storage system such as a database (MySQL, HBase, ...), a file with a specific encoding (CSV, Apache \[Parquet, Avro, ORC\], ...), or a messaging system (Apache Kafka, RabbitMQ, ...). 
-
-Flink aims to provide TableSources for common data formats and storage systems. Please have a look at the [Table Sources and Sinks]({{ site.baseurl }}/dev/table/sourceSinks.html) page for a list of supported TableSources and instructions for how to build a custom `TableSource`.
-
-A `TableSource` is registered in a `TableEnvironment` as follows:
+It is also possible to create a `TABLE` as known from relational databases from a [connector]({{ site.baseurl }}/dev/table/connect.html) declaration.
```

Comment:
```suggestion
It is also possible to create a `Table` as known from relational databases from a [connector]({{ site.baseurl }}/dev/table/connect.html) declaration.
```

?

Reply:
After second thought, I think it is more consistent to leave it with capital letters. In other places, whenever we refer to SQL terms we use capital letters.